rebasehelper/plugins/output_tools/__init__.py

```diff
@@ -51,7 +51,12 @@ def get_report_path(cls, app):
         return os.path.join(app.results_dir, REPORT + '.' + cls.EXTENSION)
 
     @classmethod
-    def prepend_results_dir_name(cls, *path_members):
+    def prepend_results_dir_name(cls, app, *path_members):
+        if app.conf.results_dir:
+            return os.path.join(app.conf.results_dir, RESULTS_DIR, *path_members)
+        elif app.conf.bugzilla_id:
```

Comment:
Are the two options mutually exclusive?

Reply:
No, both can be used at the same. This should work fine, `--results-dir` has higher priority if we look at where the results directory will be placed. But you brought up a good point. When using both of the options and passing `--results-dir` as a relative path, it's not going to work as expected.

I think the path should be converted to absolute before change directory here:
https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/cli.py#L158

Reply:
Yes, and the original directory should be saved so it can be used to determine the paths.